README.md

```diff
@@ -1,18 +1,29 @@
 # .NET Core Samples
 
 This folder contains all the sample code that is part of any topic under
-the .NET Core documentation. There are several different projects that
+the .NET documentation. There are several different projects that
 are organized in sub-folders. These sub-folders are organized similar
-to the organization of the docs for .NET Core.
+to the organization of the docs for .NET.
 
 Some of the articles will have more than one sample associated with them. 
 
 The readme.md file for each sample will refer to the article so that
 you can read more about the concepts covered in each sample.
 
+There are two classes of samples in this repository:
```

Comment:
perhaps "two classes of code" rather than two classes of samples, since one of the classes is samples